repository/RediStick-Connection-Pool.package/RsRedisConnectionPool.class/instance/createStickAt..st

```diff
@@ -5,5 +5,6 @@ createStickAt: key
 	| stick |
 	stick := RsRediStick targetUrl: self primaryUrl.
 	stick beSticky.
+	stick username: userName password: password.
```

Comment:
Avoid direct access to the variables.
`stick username: self userName password: self password`

Reply:
done a042c73   & f0cd103